plaso/parsers/defender_device.py

```diff
@@ -0,0 +1,1696 @@
+# -*- coding: utf-8 -*-
+"""Defender DeviceFileEvents CSV parser.
+
+Also see:
+https://learn.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-devicefileevents-table
+"""
+
+import csv
+import json
+import os
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.lib import errors
+from plaso.parsers import dsv_parser
+from plaso.parsers import manager
+
+
+class DefenderDeviceEventData(events.EventData):
+  """Defender DeviceFileEvents event data.
```

Comment:
DeviceFileEvents appears to be one of the advanced hunting types https://learn.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-schema-tables?view=o365-worldwide#get-schema-information-in-the-security-center